### PR TITLE
FE:  FavoriteComponent,useFavorite,model,FavoritePage,

### PR DESCRIPTION
**Pull request by NinaGhk**
component
useFavorite
model
FavoritePage

### CODE REVIEW

File: frontend/src/components/FavoriteGallery.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+type FavoriteGalleryProps = {
+    favoriteNfts:FavoriteNft[];
+}
+
+export default  function FavoriteGallery(props:FavoriteGalleryProps){
+
+    return (
+     <div className ="favoritecard">
+         {props.favoriteNfts.map((FavoriteNft) =>
```
Comment (kyunoske) [inline frontend/src/components/FavoriteGallery.tsx:14]: FavoriteNft should be {props.favoriteNfts.map((favoriteNft) =>
I am not sure if you are using your props correctly.

File: frontend/src/pages/FavoritePage.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+import {useNavigate, useParams} from "react-router-dom";
+import {FavoriteNft} from "../model/FavoriteNft";
+import FavoriteGallery from "../components/FavoriteGallery";
+
+
+
+
+type FavoriteProps ={
+    favoriteNfts: FavoriteNft[];
+    getAllFavorite: ()=>void
```
Comment (kyunoske) [inline frontend/src/pages/FavoritePage.tsx:12]: getAllFavorite should have a type. Example - getAllFavorite:(here: Here) => void;

File: frontend/src/pages/FavoritePage.tsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+            <Row className = "favorite">
+
+                <FavoriteGallery  favoriteNfts={props.favoriteNfts}/>
+            </Row>
+        </Container>
+    );
+}
+export default FavoritePage;
+
+// line 36-39       <button className = "favorite" onClick={handleClick}></button>
```
Comment (kyunoske) [inline frontend/src/pages/FavoritePage.tsx:46]: Remove all commented out code.

File: frontend/src/components/FavoriteGallery.tsx [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    return (
+     <div className ="favoritecard">
+         {props.favoriteNfts.map((favoriteNft) =>
+             <div className={"favoritecard"}>
+                 <FavoriteCard favoriteNft={favoriteNft}/>
+             </div>)}
+     </div>
+    )}
+
+        // line 17 <p>{FavoriteNft.name}</p>)}
```
Comment (AccessCoder) [inline frontend/src/components/FavoriteGallery.tsx:21]:
```suggestion
        
```

**Pull request merged**